vocab/credentials/v2/undefined-term.html

```diff
@@ -0,0 +1,68 @@
+        doJsonLd:    true,
+        editors: [{
+          name:       "Ivan Herman",
+          url:        "https://www.w3.org/People/Ivan/",
+          company:    "W3C",
+          w3cid:      7382,
+          orcid:      "0000-0003-0782-2704",
+          companyURL: "https://www.w3.org",
+          note:       "v2.0"
+        }],
+        postProcess : [remove_status_remark],
+        inlineCSS: true,
+        doRDFa: false,
+        noIDLIn: true,
+        noLegacyStyle: false
+      };
+    </script>
+  </head>
+  <body>
+    <section id="abstract">
+        <p>
+            This is the namespace document for the undefined-term properties used in a 
+            Verifiable Credential [[vc-data-model-2.0]]
+        </p>
+    </section>
+    <section id="sotd">
+        <p>
+            This namespace has been defined by the <a href="https://www.w3.org/2017/vc/WG/">W3C Verifiable Credentials Working Group</a>.
+            Comments regarding this document are welcome. Please file issues
+            directly on <a href="https://github.com/w3c/vc-data-model/issues/">GitHub</a>, or send them to
+            <a href="mailto:public-vc-comments@w3.org">public-vc-comments@w3.org</a>
+            (<a href="mailto:public-vc-comments-request@w3.org?subject=subscribe">subscribe</a>,
+            <a href="https://lists.w3.org/Archives/Public/public-vc-comments/">archives</a>).    
+        </p>
+    </section>
+    <section id="conformance"></section>
+    <section>
+        <h2>Issuer Dependent Terms</h2>
+        <p>
+            The <a href="https://www.w3.org/TR/vc-data-model-2.0/#base-context">base context</a> of a Verifiable Credential, as defined in [[vc-data-model-2.0]], provides a mapping from the standard terms used in credentials to the vocabulary items defined by the Verifiable Credentials Working Group through different vocabularies, such as the <a href="https://www.w3.org/2018/credentials/">Verifiable Credentials Vocabulary</a>. Applications using additional, user-dependent vocabularies of their own are encouraged to create their own formal vocabularies, and corresponding JSON-LD context definitions, to ensure the interoperability of the credentials they produce and/or consume.
+        </p>
+        <p>
+            There may be cases, however, when the creation of such formal vocabularies and context definitions is too much of a burden, such as for closed applications, experiments, or examples. The <a href="https://www.w3.org/ns/credentials/undefined-term/v2">context</a> has been constructed in such a way that issuers may use new, hitherto undefined terms by adding it to the Verifiable Credential, and these terms will be mapped onto URLs of the form <code>https://www.w3.org/ns/credentials/undefined-term#TERM</code>. If a conforming document does not use JSON-LD Contexts that define all terms used, it must include the `https://www.w3.org/ns/credentials/undefined-terms/v2` as the last value in the `@context` property.
```

Comment:
There is lowercase normative language such as `may` and `must` in here that we might not want to use... people might get confused by whether this is normative language or not (it's not, the main spec says these things, IIRC).

Reply:
I think lower case must/may is o.k. here. This document is not a Recommendation, and a upper case version would possibly raise eyebrows.

(Besides, I believe respec might complain...)

Reply:
Made the change in https://github.com/w3c/vc-data-model/pull/1536/commits/c41517aa88e7888f73daac698481ff6226d2c6ec